src/app/Instance/BridgeOverview/Components/OBDashboardTableView.css

```diff
@@ -0,0 +1,8 @@
+.OB-Dashboard__resource__truncated-string {
+  --pf-c-truncate--MinWidth: 0;
+  --pf-c-truncate__start--MinWidth: 0;
+}
```

Comment:
setting the minimum width of the PF Truncate component to 0. This will prevent the table from overflowing the column on lower res.